sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncContainer.java

```diff
@@ -408,6 +416,35 @@ public String getId() {
         return queryItemsInternal(new SqlQuerySpec(query), new CosmosQueryRequestOptions(), classType);
     }
 
+    /**
+     * Initializes the container by warming up the caches and connections.
+     * @return Mono of Void
+     */
+    @Beta(value = Beta.SinceVersion.V4_13_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
+    public Mono<Void> initializeContainerAsync() {
```

Comment:
none of the APIs in this class have "Async" suffix.
please remove to be consistent.

Reply:
warmup seems a better method name to me. as it might be confusing what initialize mean.

when working on a structure which have a initialize method the common sense is `initialize()` needs to be invoked. where as here it is different, this is optional warmup work so it is better to rename to be more indicative of it is warmup.

Reply:
I see "warmup" as a more confusing choice then initialize... what is the .NET choice for the API name? 
Alternative is to name the API something close to what it is meant to do, which is openConnectionsToService() ;-)

Reply:
I am fine with `openConnectionsToService()`

Reply:
as decided in scrum today going with openConnectionsAndInitCaches